smarts/core/utils/math.py

```diff
@@ -245,21 +245,27 @@ def low_pass_filter(
 
 
 def radians_to_vec(radians) -> np.ndarray:
-    """Convert a radian value to a unit directional vector."""
+    """Convert a radian value to a unit directional vector. 0 rad relates to [0, 1] with
+    counter-clockwise rotation.
+    """
```

Comment:
What does "0 rad relates to [0, 1] with ... " mean?

Reply:
It relates to the vector `[0x, 1y]`.